dev/io.openliberty.transport.http_fat/publish/files/.gitignore

```diff
@@ -0,0 +1 @@
+/CookieDate.zip
```

Comment:
what is this referencing?

Reply:
I will investigate!

Reply:
It came from the project file I cut an pasted as a start line - the http2_fat project: 
<img width="391" alt="image" src="https://user-images.githubusercontent.com/2861921/140910389-6c89c94c-2844-4ffa-9283-325d1755ad06.png">
(I will remove it)

Reply:
Gone.

Reply:
I see - looks like some cleanup is needed in the http2_fat as well. Thanks for finding this!